community.json

```diff
@@ -179,6 +179,12 @@
         "state": "notworking",
         "url": "https://github.com/YunoHost-Apps/chtickynotes_ynh"
     },
+    "codimd": {
+        "branch": "master",
+        "revision": "HEAD",
```

Comment:
I'm more in favor of a fixed commit instead of HEAD, there is no testing branch (you can use sha1 of the current HEAD revision ;-))

Reply:
That's why I asked @liberodark and @frju365 to confirm if they are ok with HEAD, which means creating a testing branch.
It's easier for packagers with HEAD.

Reply:
Normally : fixed : https://github.com/YunoHost-Apps/codimd_ynh/issues/6